.github/workflows/playwright.yml

```
@@ -34,25 +34,33 @@ jobs:
         run: npx playwright test
+      
       - uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
           path: playwright-report/
           retention-days: 30
+      
       - name: Stop server
-        run: kill $(lsof -t -i:8000)
+        run: kill $(lsof -t -i:8000)
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)